CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## v0.9.0
+* Fixed safety for sock_ctrl_msg::raw_recvmsg() and enhanced documentation
+* [[#135](https://github.com/rust-vmm/vmm-sys-util/pull/135)]: sock_ctrl_msg:
+   mark recv_with_fds as unsafe
+* [[#136[(https://github.com/rust-vmm/vmm-sys-util/pull/136)]: tempdir: fix
```

Comment:
Should we mention abut test fixes in the changelog? I don't think we used to do that.

Reply:
ok, remove it